ci/build/upload_build_info.sh

```diff
@@ -3,11 +3,14 @@
 # Cause the script to exit if a single command fails.
 set -ex
 
+chmod -R a+r /tmp/bazel_event_logs
```

Comment:
this is needed now that the mount dir is shared across multiple dockers

Reply:
why? how does multi-container mounting has anything to do with mod bits?

Reply:
ray_ci is run under forge which doesn't have sudo permissions; while the container used for running tests is run as sudo